packages/component/src/components/page-list/use-virtual-table-height.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+
+export function useVirtualTableHeight() {
+  const [virtualTableHeight, setVirtualTableHeight] = useState(0);
+  useEffect(() => {
+    const mainEl = document.querySelector('.main-container') as HTMLElement;
+    const headEl = document.querySelector('.header-container') as HTMLElement;
```

Comment:
prohibit using the constant value

Reply:
Okay, but for what reason?

Reply:
not a good engineering approach. I think we pass two `ref<HTMLElemet>` in parameters is better.

And a hook function should be pure, which also means it only depends on the inputs(parameters)

Reply:
> not a good engineering approach. I think we pass two `ref<HTMLElemet>` in parameters is better.
> 
> And a hook function should be pure, which also means it only depends on the inputs(parameters)

I've modified it.